lightning/src/ln/channelmanager.rs

```diff
@@ -3547,12 +3547,15 @@ where
 				channel_funding_txo: shutdown_res.channel_funding_txo,
 			}, None));
 
-			let funding_info = if is_manual_broadcast {
-				shutdown_res.channel_funding_txo.map(|outpoint| FundingInfo::OutPoint{ outpoint })
-			} else {
-				shutdown_res.unbroadcasted_funding_tx.map(|transaction| FundingInfo::Tx{ transaction })
-			};
-			if let Some(funding_info) = funding_info {
+			if let Some(transaction) = shutdown_res.unbroadcasted_funding_tx {
```

Comment:
Pre-existing, but I think it would be useful to have some docs on `unbroadcasted_funding_tx`. Because I assumed it would always be `None` for unsafe_manual_broadcast'd channels, but that would mean we would never hit the manual broadcast clause below so that doesn't make sense..

Reply:
Ah okay, I see the docs now